integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/SparkPropertyFacet.java

```diff
@@ -0,0 +1,26 @@
+/*
+/* Copyright 2018-2022 contributors to the OpenLineage project
```

Comment:
2023 ;-)

Reply:
I'm going to leave this one be, we will need to change all of the headers at once so we can modify the verification.